frontend/src/game/main.ts

```diff
@@ -2,6 +2,7 @@ import { Boot } from "./scenes/Boot";
 import { Collect } from "./scenes/Collect";
 import { AUTO, Game } from "phaser";
 import { Preloader } from "./scenes/Preloader";
+import { Physics } from "phaser";
```

Comment:
Is this necessary? There are no other changes to this file which leads me to believe this import is unused.